<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>887</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-radius: 40px;
background-color: rgb(211, 211, 211);
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="interface_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>811</width>
      <height>441</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 127);</string>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="Ajout">
     <attribute name="title">
      <string>Ajouter</string>
     </attribute>
     <widget class="QWidget" name="video" native="true">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>60</y>
        <width>311</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(167, 167, 167);
border-color: rgb(255, 255, 255);
border-radius: 10px;
border: 2px solid gray;</string>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>50</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
border: 2px solid gray;</string>
       </property>
       <property name="text">
        <string>Insertion video</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>250</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(167, 167, 167);
border-radius: 10px;
border: 2px solid gray;</string>
      </property>
      <property name="text">
       <string>Titre</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>290</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(167, 167, 167);
border-color: rgb(255, 255, 255);
border-radius: 10px;
border: 2px solid gray;</string>
      </property>
      <property name="text">
       <string>Description</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="titre">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>250</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
border: 2px solid gray;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>30</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(167, 167, 167);
border-radius: 10px;
border: 2px solid gray;</string>
      </property>
      <property name="text">
       <string>Type </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(167, 167, 167);
border-radius: 10px;
border: 2px solid gray;</string>
      </property>
      <property name="text">
       <string>Date</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>280</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(167, 167, 167);
border-color: rgb(255, 255, 255);
border-radius: 10px;
border: 2px solid gray;</string>
      </property>
      <property name="text">
       <string>Ajouter </string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>320</y>
        <width>81</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(167, 167, 167);
border-color: rgb(255, 255, 255);
border-radius: 10px;
border: 2px solid gray;</string>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_id">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>210</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
border: 2px solid gray;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>210</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(167, 167, 167);
border-radius: 10px;
border: 2px solid gray;</string>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="description">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>290</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
border: 2px solid gray;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="date">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
border: 2px solid gray;</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>350</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
border: 2px solid gray;</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QComboBox" name="type">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>30</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
border: 2px solid gray;</string>
      </property>
      <item>
       <property name="text">
        <string>Sport</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Economie</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Politique</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Culture</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>High Tech</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="Afficher">
     <attribute name="title">
      <string>Afficher</string>
     </attribute>
     <widget class="QTableView" name="tab_pub">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>641</width>
        <height>301</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgb(170, 170, 170);
alternate-background-color: rgb(185, 185, 185);
border-radius: 10px;
border: 2px solid gray;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="type_2">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>10</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
border: 2px solid gray;</string>
      </property>
      <item>
       <property name="text">
        <string>Sport</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Economie</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Politique</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Culture</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>High Tech</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>10</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(167, 167, 167);
border-radius: 10px;
border: 2px solid gray;</string>
      </property>
      <property name="text">
       <string>Type </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="refresh">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>360</y>
        <width>81</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
border: 2px solid gray;</string>
      </property>
      <property name="text">
       <string>Actualiser</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Modifier</string>
     </attribute>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>100</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(167, 167, 167);
border-radius: 10px;
border: 2px solid gray;</string>
      </property>
      <property name="text">
       <string> ID</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_newid">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>100</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
border: 2px solid gray;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>150</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(167, 167, 167);
border-radius: 10px;
border: 2px solid gray;</string>
      </property>
      <property name="text">
       <string>Nouveau Titre</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_newtitre">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>150</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
border: 2px solid gray;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>200</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(167, 167, 167);
border-color: rgb(255, 255, 255);
border-radius: 10px;
border: 2px solid gray;</string>
      </property>
      <property name="text">
       <string>Nouvelle Description</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_newdescription">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>200</y>
        <width>171</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
border: 2px solid gray;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>120</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(167, 167, 167);
border-radius: 10px;
border: 2px solid gray;</string>
      </property>
      <property name="text">
       <string>Nouveau Type </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>170</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(167, 167, 167);
border-radius: 10px;
border: 2px solid gray;</string>
      </property>
      <property name="text">
       <string>Nouvelle date</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_newdate">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>170</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
border: 2px solid gray;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_update">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>310</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(167, 167, 167);
border-color: rgb(255, 255, 255);
border-radius: 10px;
border: 2px solid gray;</string>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>310</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(167, 167, 167);
border-color: rgb(255, 255, 255);
border-radius: 10px;
border: 2px solid gray;</string>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
     <widget class="QComboBox" name="le_newtype">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>120</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
border: 2px solid gray;</string>
      </property>
      <item>
       <property name="text">
        <string>Sport</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Economie</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Politique</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Culture</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>High Tech</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="supprimer">
     <attribute name="title">
      <string>Supprimer</string>
     </attribute>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>100</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(167, 167, 167);
border-color: rgb(255, 255, 255);
border-radius: 10px;
border: 2px solid gray;</string>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_id_supp">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>100</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
border: 2px solid gray;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="id_supp">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>170</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(167, 167, 167);
border-color: rgb(255, 255, 255);
border-radius: 10px;
border: 2px solid gray;</string>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>340</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Franklin Gothic Heavy</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
border: 2px solid gray;</string>
      </property>
      <property name="text">
       <string>CLOUD FM</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>887</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
